libbeat/outputs/elasticsearch/client.go

```diff
@@ -70,14 +70,14 @@ type Connection struct {
 
 // Metrics that can retrieved through the expvar web interface.
 var (
-	ackedEvents            = expvar.NewInt("libbeat.es.published_and_acked_events")
-	eventsNotAcked         = expvar.NewInt("libbeat.es.published_but_not_acked_events")
-	publishEventsCallCount = expvar.NewInt("libbeat.es.call_count.PublishEvents")
-
-	statReadBytes   = expvar.NewInt("libbeat.es.publish.read_bytes")
-	statWriteBytes  = expvar.NewInt("libbeat.es.publish.write_bytes")
-	statReadErrors  = expvar.NewInt("libbeat.es.publish.read_errors")
-	statWriteErrors = expvar.NewInt("libbeat.es.publish.write_errors")
+	ackedEvents            = monitoring.NewInt(outputs.Metrics, "es.published_and_acked_events")
+	eventsNotAcked         = monitoring.NewInt(outputs.Metrics, "es.published_but_not_acked_events")
+	publishEventsCallCount = monitoring.NewInt(outputs.Metrics, "es.call_count.PublishEvents")
+
+	statReadBytes   = monitoring.NewInt(outputs.Metrics, "es.publish.read_bytes")
```

Comment:
Can we go here with the metricbeat convention? So this would be:

```
es.publish.read.bytes
es.publish.read.errors
es.publish.write.bytes
es.publish.write.errors
```